ONTOLOGY_GUIDE.md

```diff
@@ -273,137 +273,75 @@ Because we believe that it is important to ensure at least some minimal level of
 1. **Crossing domains / ranges**  - If one property is a sub of another, then the domains (respectively ranges) should not be subClasses in the opposite direction. See [testHygiene0002.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0002.sparql), [testHygiene0003.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0003.sparql)
 1. **Labels and Definitions** - Every Class and Property defined in FIBO must have an rdfs:label and a skos:definition. See [testHygiene0004.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0004.sparql)
 1. **Ontology Metadata** - Every Ontology defined in FIBO must have a rdfs:label, sm:copyright, dct:license, dct:abstract. See [testHygiene0005.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0005.sparql)
+1. **Special Characters** - The set of characters allowed in literals is limited to the alphanumeric characters, punctuation marks, and diacritic characters used in the languages FIBO uses. See [testHygiene0114.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0114.sparql)
+1. **Polysemous elements** - We should avoid using polysemous names, in particular no label can name more than one owl:Class or rdf:Property. See [testHygiene1067.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1067.sparql)
```

Comment:
2 issues here: a) I don't think many people will know what polysemy means; b) reflecting a) the hygiene test is not limited to polysemy but also catches homonymy - see https://en.wikipedia.org/wiki/Polysemy . Better IMO to talk in terms of "unique label rule" which is more comprehensible and reflects both the intent and logic

Reply:
Agreed.